guide/14-deep-learning/object-detection.ipynb

```diff
@@ -69,7 +69,7 @@
        "                        </a>\n",
```

Comment:
<p>This graph does not seem to be an output from the code ssd.lr_find(). This looks like an image added using markdown cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>